From 5aa2faac032dba15baa0dd00052daf078a269c25 Mon Sep 17 00:00:00 2001
From: Mathieu Desnoyers <mathieu.desnoyers@efficios.com>
Date: Tue, 10 Nov 2015 13:01:50 -0500
Subject: [PATCH 821/933] Fix: instrumentation: v4lv2 missing include

Starting at least with 4.3 kernel, we need to specifically include
media/videobuf2-core.h.

Signed-off-by: Mathieu Desnoyers <mathieu.desnoyers@efficios.com>
---
 drivers/staging/lttng/probes/lttng-probe-v4l2.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/staging/lttng/probes/lttng-probe-v4l2.c b/drivers/staging/lttng/probes/lttng-probe-v4l2.c
index 3b1985c..bdb38fd 100644
--- a/drivers/staging/lttng/probes/lttng-probe-v4l2.c
+++ b/drivers/staging/lttng/probes/lttng-probe-v4l2.c
@@ -23,8 +23,8 @@
 
 #include <linux/module.h>
 #include <linux/videodev2.h>
+#include <media/videobuf2-core.h>
 #include "../lttng-tracer.h"
-
 /*
  * Create the tracepoint static inlines from the kernel to validate that our
  * trace event macros match the kernel we run on.
-- 
2.0.2

